utils/query-string.js

```diff
@@ -34,6 +34,17 @@ function queryparser(query) {
 
   regex.lastIndex = 0;
   return params;
-}
+};
 
-module.exports = queryparser;
+exports.stringify = function stringify(obj) {
+  let string = '';
+  for (const prop in obj) {
+    if (string) {
+      string += '&';
+    }
+
+    string += `${encodeURIComponent(prop)}=${encodeURIComponent(obj[prop])}`;
```

Comment:
We need to encode the property name too?

Reply:
Yup. I'm not sure of the security implications on Pantheon's side, but I could easily post an extra field that with special characters in the property name.